codeSnippets/snippets/client-call-id/src/main/kotlin/com/example/Application.kt

```diff
@@ -0,0 +1,39 @@
+package com.example
+
+import io.ktor.client.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.client.plugins.callid.*
+import io.ktor.http.*
+import kotlinx.coroutines.*
+
+fun main() {
+    runBlocking {
+        val client = HttpClient(CIO) {
+            install(CallId) {
+                useCoroutineContext = true
```

Comment:
I think we should not explicitly set default values in config

Reply:
I set it to false in the next example and adjusted the docs.